bigquery_etl/view/generate_stable_views.py

```diff
@@ -222,6 +225,19 @@ def write_view_if_not_exists(target_project: str, sql_dir: Path, schema: SchemaF
         with metadata_file.open("w") as f:
             f.write(metadata_content)
 
+    # get view schema with descriptions
+    try:
+        regex = re.compile(r"CREATE OR REPLACE VIEW\n\s*[^\s]+\s*\nAS", re.IGNORECASE)
+        content = regex.sub("", target_file.read_text())
+        content += " WHERE DATE(submission_timestamp) = '2020-01-01'"
+        view_schema = Schema.from_query_file(target_file, content=content)
+
+        stable_table_schema = Schema.from_json({"fields": schema.schema})
+        view_schema.merge(stable_table_schema, add_missing_fields=False)
+        view_schema.to_yaml_file(target_dir / "schema.yaml")
+    except Exception as e:
```

Comment:
Is it possible to be more specific about what exceptions we're handling here? There are three instance of `except Exception` in this PR where we print the message and then move on, which could lead to confusing behavior where it's unclear to the user whether the exception is expected or not. And particularly for automated running, that could lead to deployment being labeled as success even if there's some issue that causes all the views to fail.

Reply:
Added some more specific exceptions where it makes sense. 
In some cases, e.g. when adding descriptions to the schema, I don't think we ever want deployment to be blocked in case of errors (e.g. dry run times out, schemas are incompatible, permissions missing). Worst case descriptions might not be available, but they don't seem very critical.